README.md

```diff
@@ -172,40 +172,60 @@ quality_report(folder_sequences = directory_sequences,
 
-Default parameters for B cell receptors in macaques: 
+Default parameters for B cell receptors in macaques and humans as meantioned above: 
 
 ```r
-# `raw_length` = 400 
+# `raw_length` = 343 # based on your expected amplicon length
 # `trim_start` = 50 
 # `trim_finish` = 409
 # `trimmed_mean_quality` = 30
-# `posvalue_probe1` = 600 
-# `posvalue_probe2` = 400 
 # `cdr3_start` = 100
 # `cdr3_end` = 150
 
 ```
 
-Default parameters for T cell receptors in mice, mus musculus: 
+Default parameters for gamma delta T cell receptors in mice (_Mus musculus_): 
 
 ```r
-# `trim_start` = 200 
+# `raw_length` = 200 
 # `trim_finish` = 250
-# `trimmed_mean_quality` = 30
 
 ```
 
 
-Default parameters for Y cell receptors in zebrafish / other species / when uncertain:
+Default parameters for when uncertain:
+
+For other species not listed or if you are uncertain, a `trimmed_mean_quality` or minimum phred quality score = 30 should be used as this returns an average of 99.9% base call accuracy for the analyzed samples.
 
 ```r
 # `trimmed_mean_quality` = 30
 
 ```
 
+If you are uncertain of your amplicon length, first check the quality metrics of your sequences using `summarise_quality`. Save the results in an object to evaluate the quality metrics columns, especially `raw.length` and `trim.finish` columns, such as the mean, min, and max values. This will help you to identify which parameters you should change.
+
+```r
+quality_control_summary <- summarise_quality(folder_sequences = directory_sequences,
+                                         secondary.peak.ratio = 0.33,
```

Comment:
remove line 211 and 212

Reply:
This code chunk runs without error or warnings, addressed in commit 8047b39466702d35773fd07e2e205a2b73dcb09e

Reply:
8047b39466702d35773fd07e2e205a2b73dcb09e

Reply:
this commit is not linked but I will assume it is solved anyway